src/Components/MovieDetails/MovieDetails.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+
+
+class MovieDetails extends Component {
+  constructor(){
+    super()
+    this.state= {
+      movie: null
+    }
+  }
+
+  componentDidMount = () =>{
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v1/movies')
+      .then(response => response.json())
+      .then(data => this.setState({movie: data.movies.find(x => x.id == this.props.match.params.movie_id)}))
+  }
+
+  render() {
+    return(
+      <div>
+      {this.state.movie &&
+        <div>
+          <h2>{this.state.movie.title}</h2>
+          <img src={this.state.movie.poster_path}/>
```

Comment:
Add in an alt prop or aria label with the movie title plus  " poster image" or "image"